requirements.txt

```diff
@@ -26,9 +26,45 @@ bitbox02==6.1.1 \
     # via hwi
 cbor==1.0.0 \
```

Comment:
You might be able to lose `cbor` - it was only present for jade in hwi, and that was upgraded there to use cbor2 (assuming you have latest cut of jade/hwi).  [ unless it is used elsewhere, of course ]

Reply:
Mhh, yes, looks like it is still in there as a transitive dependency from hwi, I am planning / hoping to upgrade to the latest hwi version soonish (to support Trezor 3s) - so will be resolved by this then I guess.